python/src/RatioOfUniforms_doc.i.in

```diff
@@ -0,0 +1,328 @@
+%feature("docstring") OT::RatioOfUniforms
+"Ratio of Uniforms simulation algorithm.
+
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Available constructors:
+    RatioOfUniforms(*dist*)
+
+    RatioOfUniforms(*logUnscaledF, rangeF*)
+    
+Parameters
+----------
+dist : :class:`~openturns.Distribution`
+    Distribution to sample.
+
+logUnscaledPDF : :class:`~openturns.Function`
+   Function :math:`\log f` where :math:`f` is proportional to the density function :math:`p` we want to sample.
+
+rangeF : :class:`~openturns.Interval`
+    Interval outside which :math:`f=0`.
+
+Notes
+-----
+This algorithm samples a multivariate distribution whose probability density probability is known up to a multiplicative factor.
```

Comment:
Please give the bibliographic references of the algorithm.

Reply:
done